pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -0,0 +1,76 @@
+package certrotationcontroller
+
+import (
+	"time"
+
+	"k8s.io/apiserver/pkg/authentication/user"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/certrotation"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+type CertRotationController struct {
+	certRotators []*certrotation.CertRotationController
+}
+
+func NewCertRotationController(
+	kubeClient kubernetes.Interface,
+	operatorClient v1helpers.StaticPodOperatorClient,
+	kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces,
+	eventRecorder events.Recorder,
+) (*CertRotationController, error) {
+	ret := &CertRotationController{}
+
+	certRotator, err := certrotation.NewCertRotationController(
+		"CSRSigningCert",
+		certrotation.SigningRotation{
+			Namespace: operatorclient.OperatorNamespace,
+			// this is not a typo, this is the signer of the signer
+			Name: "csr-signer-signer",
+			// TODO we will probably make this much longer lived
+			Validity:          1 * 8 * time.Hour,
+			RefreshPercentage: 0.5,
+			Informer:          kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets(),
+			Lister:            kubeInformersForNamespaces.InformersFor(operatorclient.OperatorNamespace).Core().V1().Secrets().Lister(),
+			Client:            kubeClient.CoreV1(),
+			EventRecorder:     eventRecorder,
+		},
+		certrotation.CABundleRotation{
+			Namespace:     operatorclient.OperatorNamespace,
+			Name:          "csr-controller-signer-ca",
```

Comment:
csr-signer-ca ?

Reply:
> csr-signer-ca

That is a ca bundle that includes the certs from the `csr-signer` so that we can rotate the signers quickly to prove we can while not disrupting trust